internal/controller/services/setup/setup_controller.go

```diff
@@ -0,0 +1,49 @@
+package setup
+
+import (
+	"context"
+	"fmt"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
+
+	"github.com/opendatahub-io/opendatahub-operator/v2/api/common"
+	sr "github.com/opendatahub-io/opendatahub-operator/v2/internal/controller/services/registry"
+	"github.com/opendatahub-io/opendatahub-operator/v2/internal/controller/setupcontroller"
+)
+
+const (
+	ServiceName = "setup"
```

Comment:
to keep the old name `setupcontroller` here ?

Reply:
done